### DISCUSSION TITLE
removing [artist] and (series) out of title

### DISCUSSION THREAD

**Discussion by strangefake** [Difegue/LANraragi#561] [Q&A] [answered]
i can't quite get how to automatically remove [artist] and (series) out of title
just like here https://i.imgur.com/366BnDT.png
since i have over 5000 "ancient chinese lithographies" editing everything manually gonna be painful

**Comment by Difegue** [comment, marked as answer]
The "Filename Parsing" plugin can do this for you automatically and edit the saved title.  
You can apply it to your 5000+ files at once through [Batch Operations.](https://sugoi.gitbook.io/lanraragi/v/dev/advanced-usage/batch-tagging)

**Reply by strangefake** [reply]
oh. thank you
